dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -59,7 +52,13 @@ public void execute(final Namespace ns, final DockerfileGitHubUtil dockerfileGit
                 GitForkBranch gitForkBranch = getGitForkBranch(image, tag, ns);
 
                 log.info("Finding Dockerfiles with the image name {}...", image);
-
+                Map<String, Boolean> orgsToIncludeInSearch = new HashMap<>();
```

Comment:
See comments about why I think this is solving the wrong problem.

Reply:
The map is being modified as expected. The modification of the map happens only when we exclude orgs recursively to reduce the search space to get around the limit that Git has of returning only 1000 search results.
For every image that is being searched for from the image tag store, the map needs to start as an empty map. Only if the search of that image returns more than 1000 results does the map get modified - which is what we want. 
However, when that image is processed completely and the run proceeds to the next image, this map should again start as an empty map and not automatically exclude the orgs that were excluded during the search of some previous image from the tag store.